gcs-sink-connector/src/main/java/io/aiven/kafka/connect/gcs/GcsSinkConfig.java

```diff
@@ -438,4 +443,15 @@ public String getGcsEndpoint() {
     public String getUserAgent() {
         return getString(GCS_USER_AGENT);
     }
+
+    public ApiTracerFactory getApiTracerFactory() {
+        if (getClass(GCS_METRICS) == null) {
+            return null;
+        }
+        try {
+            return getClass(GCS_METRICS).asSubclass(ApiTracerFactory.class).getDeclaredConstructor().newInstance();
```

Comment:
I see that not all implementations of `ApiTracerFactory` have a default constructor that you are using here. So I'm just curious what which implementation is supposed to be used here?